table-examples/units-for/output.txt

```diff
@@ -0,0 +1,2 @@
+On row 3, column "a_unit", value "None" fails because Required when a_value is filled
+On row 4, column "a_unit", value "None" fails because Required when a_value is filled
```

Comment:
Most of the text of this message is generated in code outside this PR... so it could change, if needed, but that's a larger scope.